README.md

```diff
@@ -1,313 +1,48 @@
-
-### Automated versioning
-[`setuptools_scm`](https://github.com/pypa/setuptools_scm) can be used to automatically version your package. It has been pre-configured in the `pyproject.toml` file. [`setuptools_scm` will automatically infer the version using git](https://github.com/pypa/setuptools_scm#default-versioning-scheme). To manually set a new semantic version, create a tag and make sure the tag is pushed to GitHub. Make sure you commit any changes you wish to be included in this version. E.g. to bump the version to `1.0.0`:
-
-```bash
-git add .
-git commit -m "Add new changes"
-git tag -a v1.0.0 -m "Bump to version 1.0.0"
-git push --follow-tags
-```
-
-**N.B. It is also possible to perform this step by using the [GitHub web interface or CLI](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository).**
-
-
-## GitHub actions workflow
-
-A GitHub actions workflow (`.github/workflows/test_and_deploy.yml`) has been set up to run (on each commit/PR):
-* Linting checks (pre-commit).
-* Testing (only if linting checks pass)
-* Release to PyPI (only if a git tag is present and if tests pass). Requires `TWINE_API_KEY` from PyPI to be set in repository secrets.
-
-## Documentation
-
-Software documentation is important for effectively communicating how to use the software to others as well as to your future self.
-
-If you want to include documentation in your package, make sure to respond with `1 - Yes` when prompted during the `cookiecutter` setup. This will instantiate a `docs` folder with a skeleton documentation system, that you can build upon.
-
-The documentation source files are located in the `docs/source` folder and should be written in either [reStructuredText](https://docutils.sourceforge.io/rst.html) or [markdown](https://myst-parser.readthedocs.io/en/stable/syntax/typography.html). The `index.rst` file corresponds to the main page of the documentation website. Other `.rst`  or `.md` files can be included in the main page via the `toctree` directive.
-
-The documentation is built using [Sphinx](https://www.sphinx-doc.org/en/master/) and the [PyData Sphinx Theme](https://pydata-sphinx-theme.readthedocs.io/en/latest/). The `docs/source/conf.py` file contains the `Sphinx` configuration.
-
-### Building the documentation locally
-You can build and view the documentation website locally, on your machine. To do so, run the following commands from the root of your project:
-```sh
-# Install the documentation build dependencies
-pip install -r docs/requirements.txt
-# Build the documentation
-sphinx-build docs/source docs/build
-```
-This should create a `docs/build` folder. You can view the local build by opening `docs/build/index.html` in a browser.
-To refresh the documentation, after making changes, remove the `docs/build` folder and re-run the above command:
-
-```sh
-rm -rf docs/build
-sphinx-build docs/source docs/build
-```
+[View Full Documentation →](docs/source/get_started.md)
```

Comment:
I think here, just link directly to the website (it will be python-cookiecutter.neuroinformatics.dev)